python-packages/fle_utils/django_utils/command.py

```diff
@@ -138,11 +109,27 @@ def call_command_async(cmd, *args, **kwargs):
     # and have everyone else spawn threads.
     is_osx = sys.platform == 'darwin'
     in_proc = kwargs.pop('in_proc', not is_osx)
-    #in_proc = kwargs.pop('in_proc', True)S
     
     if in_proc:
         return call_command_threaded(cmd, *args, **kwargs)
     else:
+        # MUST: Check if running on PyRun to prevent crashing the server since it doesn't have
```

Comment:
This will only affect pyrun, and the OSX team are making pretty solid assertions that this works :)